docs/scale-and-perf/SLOs.md

```diff
@@ -0,0 +1,39 @@
+# KubeVirt Scalability and Performance SLIs/SLOs
+
+This document is inspired by the [Kubernetes slos document](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md) and is meant to serve the same purpose but for KubeVirt's control plane.
+
+## KubeVirt Performance and Scale Testing
+The KubeVirt SIG-scale is looking to use two tests to define scale and perf: Burst and Steady State.
+
+The goal for a **Burst Test** is to evaluate the system's performance under heavy load when the user is most
+interested in performing an operation as quickly as possible.  A good example of a Burst workload is a sudden
+spike in demand for compute resources to handle a massive increase of users.
+
+The Burst Test measures batch VM/VMI creation latency so here's how KubeVirt tests it. The Burst Test will
+ramp up object count in a datacenter, measure, then ramp down object count in a datacenter. The measurement
+will include the Object Creation rate (objects/second), VMI Phase Transition Rate, and count Kubernetes REST API calls.
+
+The **Steady State Test** takes a different approach than the burst test in that instead of measuring how quickly
+a system can move between differant capacities, the Steady State Test focuses on how well the system can maintain
+max capacity or near max capacity.  For example, a system may create a certain number of warm resources
```

Comment:
I would say the goal is not to maintain maximum capacity, but to maintain a constant number of operations per second.

For example, 20 creates and deletes per second. The total number of objects in the system will depend on the object's lifetime...

Reply:
I'm implying that.  No matter what the delete rate is, we're measuring the creation rate.  If 20 creates per second is the max create output at the test's scale, then that's what we'll expect.